docs/en/ingest-management/migrate-beats-to-agent.asciidoc

```diff
@@ -0,0 +1,513 @@
+<<data-streams>>.
+
+You have a couple of options for migrating custom dashboards:
+
+* (Recommended) Recreate the custom dashboards based on the new data streams.
+
+* <<create-index-aliases,Create index aliases to point to the new data streams>>
+and continue using custom dashboards.
+
+[discrete]
+[[create-index-aliases]]
+=== Create index aliases to point to data streams
+
+You may want to continue using older custom dashboards if the dashboards
+installed with an integration are not adequate. To do this, use index aliases
+to feed data streams into your existing custom dashboards.
+ 
+For example, custom dashboards that point to `filebeat-` or `metricbeat-` can be
+aliased to use the equivalent data streams, `logs-` and `metrics-`.
+
+To use aliases:
+
+. Add a `filebeat` alias to the `logs-` data stream. For example:
++
+[source,json]
+----
+POST _aliases
+{
+  "actions": [
+    {
+      "add": {
+        "index": "logs-*",
+        "alias": "filebeat"
+      }
+    }
+ 
+ ]
+}
+----
+
+. Add a `metribeat` alias to the `metrics-` data stream.
++
+[source,json]
+----
+POST _aliases
+{
+  "actions": [
+    {
+      "add": {
+        "index": "metrics-*",
+        "alias": "metricbeat"
+      }
+    }
+ ]
+}
+----
+
+IMPORTANT: These aliases must be added to both the index template and existing
+indices.
+
+Note that custom dashboards will show duplicated data until you remove {beats}
+from your hosts.
+
+For more information, see the {ref}/aliases.html[Aliases documentation].
+
+[discrete]
+== Migrate index lifecycle policies
+
+Index lifecycle management (ILM) policies in {es} enable you to manage indices
+according to your performance, resiliency, and retention requirements. To learn
+more about ILM, refer to the
+{ref}/index-lifecycle-management.html[ILM documentation].
+
+ILM is configured by default in {beats} (version 7.0 and later) and in {agent}
+(all versions). To view the index lifecycle policies defined in {es}, go to
+*Management > Index Lifecycle Policies*.
+
+[role="screenshot"]
+image::images/migration-index-lifecycle-policies.png[Screen showing how to add a processor to an integration policy]
+
+If you used ILM with {beats}, you'll see index lifecycle policies like
+*filebeat* and *metricbeat* in the list. After migrating to {agent}, you'll see
+polices named *logs* and *metrics*, which encapsulate the ILM policies for all
+`logs-*` and `metrics-*` index templates.
+
+When you migrate from {beats} to {agent}, you have a couple of options for
+migrating index policy settings:
+
+* *Modify the newly created index lifecycle policies (recommended).* As
+mentioned earlier, ILM is enabled by default when the {agent} is installed.
+Index lifecycle policies are created and added to the index templates for
+data streams created by integrations.
++
+If you have existing index lifecycle policies for {beats}, it's highly
+recommended that you modify the lifecycle policies for {agent} to match your
+previous policy. To do this:
```

Comment:
Does this work? IIRC, ILM changes aren't yet persisted through Elastic Agent upgrades

Reply:
I'll check into that. Surely we don't expect users to update their index lifecycle policies every time they upgrade Elastic Agent.

Reply:
Ok then it sounds like the documented procedure is correct. I can always do a follow-on PR if I hear elsewise.

Reply:
> I believe you can change the preloaded `logs` and `metrics` ILM policies without having them overwritten

So a user can change these without them getting overwritten (ILM policies only have a put-it-there-if-it-doesn't-exist behavior), however, they are currently marked as `managed: true`, which means that Kibana will dissuade (prevent? I'm not 100% sure) a user from changing them.

Reply:
I was able to modify the metrics and logs policies in Kibana (though of course I didn't try to change all the settings 😄 :). I did see a message describing the impact, which I think is appropriate: 

"You are editing an existing policy. Any changes you make will affect 16 linked index templates that are attached to this policy. Alternatively, you can save these changes in a new policy."